### ISSUE TITLE
Not able to invalidate keys that previously return `null` as value.

### ISSUE THREAD

**Issue by pobidowski** [Asjas/prisma-redis-middleware#272]
**Describe the bug**
Not able to invalidate keys that previously return `null` as value.

**To Reproduce**
1. Try to make any `findFirst` query in Prisma that returns null (start caching)
2. Create a record that matches `findFirst` from point 1.
3. Invalidation is not picking up previously created key, because `references` arg was null.
4. Try to make the same `findFirst` query. Should return record from point 2, but instead of that returns `null`.

**Expected behavior**
Invalidation of `null` values works.

**Additional context**
The problem is with function:
```
references: ({ params }: { params: MiddlewareParams }, key: string, result: Result) => {
   return result ? [`${params.model}~${key}`] : null;
},
```

When the `result` is null the references returns also `null` and invalidation function cannot pick up this key and cannot remove it. After changing this part of the code to:
```
references: ({ params }: { params: MiddlewareParams }, key: string, result: Result) => {
   return [`${params.model}~${key}`];
},
```
It worked.

**Comment by pobidowski** [comment]
@Asjas if you are happy with this fix I can create a PR

**Comment by Asjas** [comment]
Yeah 👍

**Comment by Asjas** [comment]
Fixed in v4.2.1 - thanks 🙂
